### ISSUE TITLE
Translation: pre-workshop/pre-workshop-ja.md

### ISSUE THREAD

**Issue by joelnitta** [swcarpentry-ja/assessment-archives#1]
Translation: `pre-workshop/pre-workshop-ja.md`
-----

* Please translate the content [of this file](https://github.com/swcarpentry-ja/assessment-archives/blob/master/pre-workshop/pre-workshop-ja.md) into Japanese.
* You don't need to translate the country list (it will be translated automatically with code)
* Please add a comment to the issue **before** you start translating, so other people know that you are translating.
* It is fine to comment that you will translate part of the file (it is not necessary to translate it all at once).
* Once you have finished translating, send a pull request and reference this issue.

-----
* [このフィアる](https://github.com/swcarpentry-ja/assessment-archives/blob/master/pre-workshop/pre-workshop-ja.md)の内容を日本語に翻訳してください。
* 国のリストは翻訳する必要はありません（コードで自動的に翻訳されます）。
* あなたが翻訳し始めたと分かるように、翻訳を**始める前**このイシューにコメントを書いてください。
* コメントでこのファイルの一部を翻訳しますと書いても大丈夫です（全部一気に翻訳する必要はありません）。
* 翻訳が終わったら、プルリクエストを開いて、このイシューを参照・リンクして下さい。

**Comment by masamiy** [comment]
I take it.

**Comment by joelnitta** [comment]
Wonderful, thank you!

**Comment by masamiy** [comment]
My clone does not have pre-workshop-ja.md. What shall I do?

**Comment by joelnitta** [comment]
what does `git log -n 1` say? (make sure your working directory is your clone of the repo)

**Comment by masamiy** [comment]
not a git repository (or any of the parent directories)... Mmm... I clone at my 'translation' directory and I ran the command you mentioned at that directory.

**Comment by joelnitta** [comment]
It sounds like you did not change directories into the repo.
Try `cd assessment-archives` first, then `git log -n 1`

**Comment by masamiy** [comment]
Better!
commit effefb152f5e1139c2458ce7c8e2dd5dd0b5acff (HEAD -> master, origin/master, origin/HEAD)
Author: GitHub Action <action@github.com>
Date:   Fri Mar 5 17:35:12 2021 +0000

    [ci-skip] add latest version

**Comment by joelnitta** [comment]
OK! Next what does `git remote -v` say?

**Comment by masamiy** [comment]
origin	https://github.com/masamiy/assessment-archives.git (fetch)
origin	https://github.com/masamiy/assessment-archives.git (push)

**Comment by joelnitta** [comment]
It looks like you forked the wrong repo: your fork is a copy of https://github.com/carpentries/assessment-archives (the main carpentries repo, not this one where we have our translation).
I recommend deleting your fork (online) and your clone (locally), then forking **this repo** (https://github.com/swcarpentry-ja/assessment-archives), and going from there.

**Comment by masamiy** [comment]
I understand why it happened - When forking, Git tells me I have 'carpentries/assessment-archives forked already. I only have a choice to fork again to a different account and I chose masamiy, but it forked the one I did not want :(  Is there a way to just download the file to translate and translate if I cannot fork?

**Comment by joelnitta** [comment]
That's odd... I just tried forking this repo myself and had no problems.
I looked at your account and I don't see any `assessment-archives` repos, so I think it should work.
Anyways, if that doesn't work, you can download the raw file from here: https://raw.githubusercontent.com/swcarpentry-ja/assessment-archives/master/pre-workshop/pre-workshop-ja.md

**Comment by TomKellyGenetics** [comment]
@masamiy if you're still having troubles, no need to clone a fresh copy. You can `pull` updates in as follows.

```
# change wd
cd assessment-archives
# add remote to organisation repo
git remote add swc-ja git@github.com:swcarpentry-ja/assessment-archives.git
# pull updated to our version (e.g., PR #3)
git pull swc-ja master
```

To push to your own click "fork" https://github.com/swcarpentry-ja/assessment-archives/ and select your name to create https://github.com/masamiy/assessment-archives
```
# add your account locally
git remote add masamiy git@github.com:masamiy/assessment-archives.git
# push your changes to your account
git push masamiy master
```

Then you can submit a pull request here:
https://github.com/swcarpentry-ja/assessment-archives/compare/master...masamiy:master

Some of this will be familiar but just in case you're stuck here's a full walkthough.

**Comment by masamiy** [comment]
Thanks for your care, @TomKellyGenetics! Whatever reason, I finally managed to do whatever I needed to do. Let's hope I can submit a pull request :D

**Comment by TomKellyGenetics** [comment]
Well done! I imagine that's the worst issues resolved. Pushing and submitting a PR should similar to how you've done for R lessons 👍

**Comment by joelnitta** [comment]
Resolved by #6
